pyproject.toml

```diff
@@ -43,6 +42,7 @@ urllib3 = "==1.25.10"
 werkzeug = "==1.0.1"
 wheel = "==0.34.*,>=0.34.2"
 wrapt = "==1.12.1"
+protobuf = "^3.13.0"
```

Comment:
This file should be trimmed down so it doesn't include nested dependencies.

Also things like pytest, selenium, pylint belong in the dev dependencies section.